.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.11"
+  - 4
+  - 6
```

Comment:
None of the dependencies test 8. `twig.js` tests 4 and 6. Since Twig is what we care about it makes sense to match its tests